README.md

```diff
@@ -1 +1,190 @@
+
+You can find more information about the ticket reservation system in this [article](https://db-blog.web.cern.ch/blog/viktor-kozlovszky/2019-10-open-days-reservation-systems-high-level-overview-2019).
+
+The content is orginized into different folders. You will find the description for each folder in this ReadMe. 
+
+Overview about the folder structure:
+* **application**: The ticket reservation application which served as the user interface for reserving the Open Days tickets. The application works by making API calls against the [back-end server](https://github.com/cerndb/open-days-registration-application-back-end) (business layer). Please note that in order to use or test the functionalities of the application you need to have a back-end server up and running which recieves and processes the API calls. The back-end project for can be found in this [repository](). For more detailed information check out the [ticket reservation application section](#Ticket-reservation-application).
+* **application-welcome**: This is a language selector page, it was hosted at the root level of the web server. Based on the language selection choice it forwards the user to the selected language path forexample: "/en/index.html" or "/fr/index.html". For more detailed information check out the [langauge preference selector application section](#Langauge-preference-selector-application).
+* **containerize**: This folder contains the helper files to build a container which hosts the two applications. For more detailed information check out the [container approach section](#Container-approach).
+
+# Collaborators
+
+The final application is the collaborated work of the following people:
+* [Nuno Guilherme Matos de Barros](https://github.com/ngmatos)
+* [Thomas Løkkeborg](https://github.com/tholok97)
+* [Viktor Kozlovszky](https://github.com/kviktorman)
+
+# Container approach
+
+**environment variables**
+
+In the docker file you need to set the following variables to their default values. Note that if you use Kubernetes for hosting the container you can override the default values in the deployment templates.
+
+| variable name | description |
+| --- | --- |
+| REGISTRATION_API_URL | The URL of the back-end application server. You will see the following error message if the back-end server is not available as you load the front-end page "Network connection failure, please try the last action again." |
+| RECAPTCHA_SITE_KEY | site key for google re-captcha. In case you don't want to use it set to any string value and set the RECAPTCHA_ENABLED to false. |
+| RECAPTCHA_ENABLED | true or false value  |
+| PIWIK_URL | The PIWIK server/service URL. |
```

Comment:
I am not sure about the popularity of piwik. Perhaps it would be useful to add a link or just add between brackets something like (web analytics application + a link https://matomo.org/blog/2018/01/piwik-is-now-matomo/)

Reply:
I have just read the reference about piwik/matomo below, forget about above comment

Reply:
Ok